pkg/applyset/prune.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2025 The Kube Resource Orchestrator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Applylib has code copied over from:
+//   - kubectl pkg/cmd/apply/applyset.go
```

Comment:
I believe we also need to bring over the copyright headers from the forked code in each file to properly attribute the code.

Reply:
Added  `Copyright 2023 The Kubernetes Authors.`

Rest of the license is common.